### PR TITLE
long to int

### PR DESCRIPTION
**Pull request by FranciscoSilvaMgLPT**

### CODE REVIEW

File: backend-files/src/main/java/com/example/punished/expensetracker/service/UserService.java [hunk 1]
```diff
@@ -27,14 +27,14 @@ public User addUser(User user) {
                     .build();
         }
 
-        public User getUserById(final Long userId) {
-            Optional<User> userAux = repository.findById(userId);
+        public User getUserById(final int userId) {
+            Optional<User> userAux = repository.findById((long) userId);
```
Comment (PedroBarge) [inline backend-files/src/main/java/com/example/punished/expensetracker/service/UserService.java:31]: you can use getReferenceById. 
and remove Optional.
and create Exception UserNotFound

File: backend-files/src/main/java/com/example/punished/expensetracker/service/UserService.java [hunk 2]
```diff
@@ -27,14 +27,14 @@ public User addUser(User user) {
...
             return User.builder()
                     .username(userAux.get().getUsername())
                     .build();
         }
-        public void deleteUser(final Long userId) {
-            Optional<User> user = repository.findById(userId);
+        public void deleteUser(final int userId) {
+            Optional<User> user = repository.findById((long) userId);
```
Comment (PedroBarge) [inline backend-files/src/main/java/com/example/punished/expensetracker/service/UserService.java:37]: same here

File: backend-files/src/main/java/com/example/punished/expensetracker/service/UserService.java (hunk 1 above)
Comment (FranciscoSilvaMgLPT) [reply backend-files/src/main/java/com/example/punished/expensetracker/service/UserService.java:31]: getreferencebyid or findbyid, I think it doesn't matter. 
The optional is for the case where they provide a nonexistent ID. 
Exceptions are still coming in the next update 😜.

**Review by fabiofilipesilva — APPROVED** [review]
mds

**Review by PedroBarge — APPROVED** [review]
ok

**Pull request merged**